doc/classes/NodePath.xml

```diff
@@ -39,74 +47,81 @@
-				The "subnames" optionally included after the path to the target node can point to resources or properties, and can also be nested.
-				Examples of valid NodePaths (assuming that those nodes exist and have the referenced resources or properties):
+				Constructs a [NodePath] from a [String]. The created path is absolute if prefixed with a slash (see [method is_absolute]).
+				The "subnames" optionally included after the path to the target node can point to properties, and can also be nested.
+				Examples of strings that could be node paths:
 				[codeblock]
 				# Points to the Sprite2D node.
-				"Path2D/PathFollow2D/Sprite2D"
+				"Level/RigidBody2D/Sprite2D"
+
 				# Points to the Sprite2D node and its "texture" resource.
-				# get_node() would retrieve "Sprite2D", while get_node_and_resource()
+				# get_node() would retrieve the Sprite2D, while get_node_and_resource()
 				# would retrieve both the Sprite2D node and the "texture" resource.
-				"Path2D/PathFollow2D/Sprite2D:texture"
+				"Level/RigidBody2D/Sprite2D:texture"
+
 				# Points to the Sprite2D node and its "position" property.
-				"Path2D/PathFollow2D/Sprite2D:position"
+				"Level/RigidBody2D/Sprite2D:position"
+
 				# Points to the Sprite2D node and the "x" component of its "position" property.
-				"Path2D/PathFollow2D/Sprite2D:position:x"
-				# Absolute path (from "root")
-				"/root/Level/Path2D"
+				"Level/RigidBody2D/Sprite2D:position:x"
+
+				# Points to the RigidBody2D node as an absolute path beginning from the SceneTree.
+				"/root/Level/RigidBody2D"
 				[/codeblock]
+				[b]Note:[/b] In GDScript, it's also possible to convert a constant string into a node path by prefixing it with [code]^[/code]. [code]^"path/to/node"[/code] is equivalent to [code]NodePath("path/to/node")[/code].
 			</description>
 		</constructor>
 	</constructors>
 	<methods>
 		<method name="get_as_property_path" qualifiers="const">
 			<return type="NodePath" />
 			<description>
-				Returns a node path with a colon character ([code]:[/code]) prepended, transforming it to a pure property path with no node name (defaults to resolving from the current node).
+				Returns a copy of this node path with a colon character ([code]:[/code]) prefixed, transforming it to a pure property path with no node names (relative to the current node).
 				[codeblocks]
 				[gdscript]
-				# This will be parsed as a node path to the "x" property in the "position" node.
-				var node_path = NodePath("position:x")
-				# This will be parsed as a node path to the "x" component of the "position" property in the current node.
+				# node_path points to the "x" property of the child node named "position".
+				var node_path = ^"position:x"
+
+				# property_path points to the "position" in the "x" axis of this node.
 				var property_path = node_path.get_as_property_path()
-				print(property_path) # :position:x
+				print(property_path) # Prints ":position:x"
```

Comment:
I'm sorry, I don't think I agree with any of the suggestions for the trailing period. It seems like the general rule in the documentation is to append it, unless the comment is the most common "Prints _", "_ is _" "Returns _".
You can see this pretty prevalently in the String documentation

Reply:
It's part of the comment style, and is used extensively in the documentation elsewhere, it's enforced in code for even short comments so I'd say it applies here too, the fact that it hasn't been enforced in the last doesn't really change this IMO, which I'd say explains the exceptions

Reply:
That is a shame. As enforced as it is, I believe readability is sacrificed here in favour of consistency, especially in the built-in class reference. I can apply the changes, somewhat reluctantly.

Reply:
It's up to you 🙂, I don't see how this harms the readability though

Reply:
It's noisy. If it were consistent across the board, it may not be a big deal for String-like types. But as for number-types, there can be a lot of commas & periods already, and the trailing period does not add any useful information.

```
red.lerp(aqua, 0.2) # Returns Color(0.8, 0.2, 0.16).
var absolute = rect.abs() # absolute is Rect2(-75.0, -25.0, 100.0, 50.0)
rect = rect.expand(Vector2i(10, 0)) # rect is Rect2i(0, 0, 10, 2)
rect = rect.expand(Vector2i(-5, 5)) # rect is Rect2i(-5, 0, 10, 5)
```

This is why I am personally fond of this inconsistency. I need to discuss it somewhere else, someday. It's not the place here. For now, I will apply the suggestions.